### ISSUE TITLE
Build generates too many JS files

### ISSUE THREAD

**Issue by chico-malo** [parcel-bundler/parcel#3579]
As the title says, I really didn't think of a better way. The packaged JS file has 2mb, which makes the website need to download for 20 seconds to open.

**Comment by mischnic** [comment]
> too many JS files

> packaged JS file has 2mb

These are two independent problems, and we can't help you without more details.

**Comment by chico-malo** [comment]
> > too many JS files
> 
> > packaged JS file has 2mb
> 
> These are two independent problems, and we can't help you without more details.

Thank you very much. Let me rephrase my question
1. Parcel build has no configuration to compress the volume of js files.In my tests, the project also had a 2mb package with few dependencies.
2. Another question is whether using CDN reference dependency will speed up access.
![image](https://user-images.githubusercontent.com/26463204/65824291-f7b12980-e298-11e9-89c8-64ef1e121850.png)
I really want to find the reason, my server bandwidth is 1mb...

**Comment by mischnic** [comment]
> Parcel build has no configuration to compress the volume of js files.In my tests, the project also had a 2mb package with few dependencies.

One option would be to use the `--experimental-scope-hoisting` flag with `parcel build`.
Can you share that project?

> other question is whether using CDN reference dependency will speed up access.

You'll have to find out yourself. If the users connection to the CDN is faster than to your server, yes.
